GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1484,15 +1484,22 @@ void DoCommit()
 
                     ScriptManager.RunEventScripts(this, ScriptEvent.AfterCommit);
 
-                    Message.Text = string.Empty; // Message.Text has been used and stored
+                    // Message.Text has been used and stored
                     _commitMessageManager.ResetCommitMessage();
                     CommitKind = CommitKind.Normal;
 
```

Comment:
Remove this empty line, please, in order to indicate - at least a little - that `Message.Text = string.Empty` belongs to the comment above the block.

Reply:
Done!